src/content/lodash/object/merge/vanilla.js

```diff
@@ -1,52 +1,45 @@
+const isObject = data => data && Object.prototype.toString.call(data) === '[object Object]'
+const isArray = data => Array.isArray(data)
+
 function merge (...args) {
-  if (!args || !args.length) {
-    return {}
-  }
+  const [firstArg, secondArg] = args
```

Comment:
The lodash method _.merge supports merging of more that 2 arguments. For example -_.merge({a:1}, {b:2}, {c: 3}).

Reply:
If I remove that line the test would fail atm.

I was wondering if this would be better:

```
const [firstArg, ...otherArgs] = args
if (typeof firstArg === 'undefined') { return merge(...otherArgs) }
```

I didn't try it, but it seems it could be at least on the right track? 
I'm happy to come up with a more complex example to prove the use case you mentioned tho)

Reply:
I was totally wrong in my previous assumpions.
I added more tests cases, with 3rd arguments with arrays and nested objects and sorted it out. 
I also refactored it a bit. What do you thin? :)